### PR TITLE
[cli] Convert `vc projects` to TypeScript

### PR DESCRIPTION
**Pull request by markozxuu**
Yesterday, I spoke with @TooTallNate about of why the command `projects` not appears in the option to the `help` command. We conclude that the team very probably forgot to add it.  This PR focuses on the covert `vc projects` to TypeScript and also added the command to the options of the help :)

#### Tests
- [ ] The code changed/added as part of this PR has been covered with tests
- [x] All tests pass locally with `yarn test-unit`

#### Code Review

- [x] This PR has a concise title and thorough description useful to a reviewer
- [ ] Issue from task tracker has a link to this PR

### CODE REVIEW

File: packages/cli/src/commands/deploy/args.ts [hunk 1]
```diff
@@ -16,6 +16,7 @@ export const help = () => `
       env                              Manages the Environment Variables for your current Project
       init                 [example]   Initialize an example project
       ls | list            [app]       Lists deployments
+      projects                         Show all projects in the selected team/user
```
Comment (styfle) [inline packages/cli/src/commands/deploy/args.ts]: This should go under Advanced just above `domains` and the description should be `Manages your Projects`

Comment (styfle) [reply packages/cli/src/commands/deploy/args.ts:19]: Also this should go under Advanced just about `domains`

File: packages/cli/src/commands/deploy/args.ts [hunk 2]
```diff
@@ -27,6 +27,7 @@ export const help = () => `
 
       rm | remove          [id]        Removes a deployment
       domains              [name]      Manages your domain names
+      proejcts                         Manages your Projects
```
Comment (styfle) [inline packages/cli/src/commands/deploy/args.ts]:
```suggestion
      projects                         Manages your Projects
```

**Review by styfle — APPROVED** [review]
Thanks!

**Review by TooTallNate — APPROVED** [review]

**Pull request merged**